presto-main/src/test/java/com/facebook/presto/sql/planner/assertions/PlanMatchPattern.java

```diff
@@ -260,13 +323,19 @@ public boolean equals(Object obj)
             }
             FunctionCall o = (FunctionCall) obj;
             return Objects.equals(getName(), o.getName()) &&
-                    Objects.equals(getArguments(), o.getArguments());
+                    Objects.equals(getArguments(), o.getArguments()) &&
+                    /*
```

Comment:
Can't un-optional it; FunctionCall represents functions that legitimately don't have a window (i.e. non-window functions). I like the checkState suggestion though.